### PR TITLE
Add decisions table

### PR DESCRIPTION
**Pull request by lowellrex**
Connects #7148.

We will be needing the `decisions` table to hold information for AMA appeals we will be getting from the IDT API. This PR creates that table but does not yet include any logic around putting things into that table.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -637,6 +647,13 @@
     t.index ["appeal_type", "appeal_id"], name: "index_special_issue_lists_on_appeal_type_and_appeal_id"
   end
 
+  create_table "staff_field_for_organizations", force: :cascade do |t|
```
Comment (mdbenjam) [inline db/schema.rb]: I think your DB already had the staff field table in it so it put it in schema.

File: db/migrate/20180926143801_create_decisions.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class CreateDecisions < ActiveRecord::Migration[5.1]
+  def change
+    create_table :decisions do |t|
+      t.belongs_to :request_issue
```
Comment (amprokop) [inline db/migrate/20180926143801_create_decisions.rb]: Why "request_issue", instead of "appeal"?

Comment (lowellrex) [reply db/migrate/20180926143801_create_decisions.rb]: Copy/paste error! Great catch!

File: db/schema.rb [hunk 3]
```diff
@@ -187,6 +187,16 @@
     t.integer "request_issue_id"
   end
 
+  create_table "decisions", force: :cascade do |t|
+    t.bigint "request_issue_id"
+    t.string "citation_number"
```
Comment (amprokop) [inline db/schema.rb]: potentially we could add a uniqueness constraint on this now, or do it when we implement the validation logic

Comment (lowellrex) [reply db/schema.rb]: "This" being the citation number?

Comment (mdbenjam) [reply db/schema.rb]: That's a good point, we should put the unique constraint on the citation number on the DB level.

**Pull request merged**